site/class-1-load/zezavdh/zezavdh/dinners.txt

```diff
@@ -0,0 +1,3 @@
+2017-09-25,kip
```

Comment:
You have a directory in your directory! Could you remove this branch, and try again? This time, only upload the files (not a directory)!